src/app/shared/error404/error404.component.html

```diff
@@ -0,0 +1,9 @@
+<div class="error">
+  <div class="error__text">
+    <p class="font-heading-1">404</p>
```

Comment:
dlaczego skoro to heading nie używamy tagu `h1`

Reply:
Poprawione